### PR TITLE
[MM-T1791] Cypress Test for Permalink 'Jump to' in Search

### PR DESCRIPTION
**Pull request by djanda97**
#### Summary

Adds an E2E test for Permalink 'Jump to' in Search.

#### Ticket Link

MM-T1791

### CODE REVIEW

File: e2e/cypress/integration/toast/permalink_jump_to_spec.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            testTeam = team;
+        });
+    });
+
+    it('MM-T1791 Permalink \'Jump to\' in Search', () => {
+        cy.visit(`/${testTeam.name}/channels/town-square`);
+
+        // # Search for a term e.g.test
+        const searchTerm = 'test';
+        cy.get('#post_textbox').type(searchTerm).type('{enter}');
```
Comment (saturninoabril) [inline e2e/cypress/integration/toast/permalink_jump_to_spec.js:27]: Please use `cy.postMessage` to take advantage of internal wait to textbox.
```suggestion
        cy.postMessage(searchTerm);
```

File: e2e/cypress/integration/toast/permalink_jump_to_spec.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        cy.get('#searchBox').type(searchTerm).type('{enter}');
+
+        // # Click on Jump to link in search results
+        cy.get('.search-item__jump').first().click();
+
+        cy.getLastPostId().then((postId) => {
+            // # Jump to link opens on main channel view
+            cy.get('#channelHeaderInfo').should('be.visible').and('contain', 'Town Square');
```
Comment (saturninoabril) [inline e2e/cypress/integration/toast/permalink_jump_to_spec.js:35]: Please add permalink URL check
```suggestion
            cy.url().should('include', `/${testTeam.name}/channels/town-square/${postId}`);
            cy.get('#channelHeaderInfo').should('be.visible').and('contain', 'Town Square');
```

File: e2e/cypress/integration/toast/permalink_jump_to_spec.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+            // # Post is highlighted and fads within 6 sec.
```
Comment (saturninoabril) [inline e2e/cypress/integration/toast/permalink_jump_to_spec.js:37]: Typo on our test case which I've recently corrected.
```suggestion
            // # Post is highlighted and fade within 6 sec.
```

File: e2e/cypress/integration/toast/permalink_jump_to_spec.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+            cy.get(`#post_${postId}`).should('have.css', 'animation-duration', '1s');
+
+            cy.get(`#post_${postId}`).should('have.css', 'animation-delay', '5s');
```
Comment (saturninoabril) [inline e2e/cypress/integration/toast/permalink_jump_to_spec.js:40]: Please add check to `post--highlight` class where it disappeared after some time.
```suggestion
            cy.get(`#post_${postId}`).
                should('have.css', 'animation-duration', '1s').
                and('have.css', 'animation-delay', '5s').
                and('have.class', 'post--highlight');
            cy.get(`#post_${postId}`).should('not.have.class', 'post--highlight');
```

File: e2e/cypress/integration/toast/permalink_jump_to_spec.js [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+            // # URL changes to channel url
+            cy.url().should('include', `/${testTeam.name}/channels/town-square`);
```
Comment (saturninoabril) [inline e2e/cypress/integration/toast/permalink_jump_to_spec.js:43]: Please add check that the post ID is no longer in the URL.
```suggestion
            cy.url().should('include', `/${testTeam.name}/channels/town-square`).and('not.include', postId);
```

**Comment by djanda97** [comment]
@saturninoabril Updated to include the suggestions you made!

File: e2e/cypress/integration/toast/permalink_jump_to_spec.js [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+// ***************************************************************
+// # Indicates a test step (e.g. # Go to a page)
+// [*] indicates an assertion (e.g. * Check the title)
+// Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @toast
+
+describe('Permalink', () => {
```
Comment (srkgupta) [inline e2e/cypress/integration/toast/permalink_jump_to_spec.js:12]: Can you please change this to `Toast` like the other test.

**Pull request merged**